### PR TITLE
Laboratory work #2, Anastasia Gubarenko - 21FPL2

### PR DESCRIPTION
**Pull request by cybern00dle**

### CODE REVIEW

File: lab_2_keywords_cooccurrence/main.py [hunk 1]
```diff
@@ -3,12 +3,75 @@
...
+    Checks if the object is empty if necessary.
+    """
+    if not isinstance(check_object, check_type):
+        return False
+    if not check_object and not can_be_empty:
+        return False
+    return True
+
+
+def check_list(check_object: Any, token_type: Type[Any], can_be_empty: bool = True) -> bool:
```
Comment (artyomtugaryov) [inline lab_2_keywords_cooccurrence/main.py:24]: import from the first lab

File: lab_2_keywords_cooccurrence/main.py [hunk 2]
```diff
@@ -3,12 +3,75 @@
...
+    return True
+
+
+def check_int(check_object: Any) -> bool:
+    """
+    Checks if an object is a positive integer.
+    """
+    if not isinstance(check_object, int) or isinstance(check_object, bool):
+        return False
+    if not check_object > 0:
```
Comment (artyomtugaryov) [inline lab_2_keywords_cooccurrence/main.py:70]: what does this check do? |
It is too difficult to understand. 
Can you simplify?

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

File: lab_2_keywords_cooccurrence/main.py (hunk 1 above)
Comment (cybern00dle) [reply lab_2_keywords_cooccurrence/main.py]: done, imported

File: lab_2_keywords_cooccurrence/main.py (hunk 2 above)
Comment (cybern00dle) [reply lab_2_keywords_cooccurrence/main.py]: removed it (it isn't needed yet), will simply import from the first lab if needed

**Review by artyomtugaryov — APPROVED** [review]

**Pull request closed without merging**